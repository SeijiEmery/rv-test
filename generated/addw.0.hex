0000000    b3  84  05  00  33  09  06  00  b3  89  06  00  33  0a  07  00
0000020    93  02  10  00  13  03  f0  ff  9b  85  15  00  13  06  16  00
0000040    9b  86  f6  ff  13  07  f7  ff  bb  84  54  00  33  09  59  00
0000060    bb  89  69  00  33  0a  6a  00  13  00  00  00  13  00  00  00
0000100    13  00  00  00  13  00  00  00  13  00  00  00  73  00  10  00
0000120
