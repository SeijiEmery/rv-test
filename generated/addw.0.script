load /x 0 generated/addw.0.hex
writereg 11 4294967295
writereg 12 4294967295
writereg 13 0
writereg 14 0
run 0 100000
readreg 11
readreg 12
readreg 13
readreg 14
readreg 9
readreg 18
readreg 19
readreg 20