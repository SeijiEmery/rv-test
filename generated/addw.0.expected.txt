tests/addw.test.s:2
R11 = 0
R12 = 4294967296
R13 = -1
R14 = -1
R9 = 0
R18 = 4294967296
R19 = -1
R20 = -1
